addons/html_editor/static/src/fields/html_field.js

```diff
@@ -351,3 +351,22 @@ export const htmlField = {
 };
 
 registry.category("fields").add("html", htmlField, { force: true });
+
+export function getHtmlFieldMetadata(content) {
+    const historyStepId = content.match(/data-last-history-steps\s*=\s*"([^"]+)"/)?.[1];
+    const version = content.match(/data-oe-version\s*=\s*"([^"]+)"/)?.[1];
+    return { historyStepId, version };
```

Comment:
I think it would make more sense to name the variable `lastHistorySteps` to match the name of the attribute.